include/nil/blueprint/manifest.hpp

```diff
@@ -658,7 +658,7 @@ namespace nil {
                 for (auto it = other_set->set.lower_bound(new_start); it !=  other_set->set.end(); it++) {
                     new_set.insert(*it);
                 }
-                for (auto it = new_start + (step - new_start % step) % step; it < finish; it += step) {
+                for (auto it = new_start + (step - new_start % step) % step; it < std::uint32_t(finish); it += step) {
```

Comment:
Same here, looks like these things are related to size_t, but you use uint32_t instead, which is probably the same on most compilers.